packages/website/docs/creating-a-renderer.md

```diff
@@ -274,7 +274,7 @@ function Autocomplete() {
 
   React.useEffect(() => {
     if (!(formRef.current && panelRef.current && inputRef.current)) {
-      return;
+      return = true;
```

Comment:
hmm, useEffect allows `return`, while what you're suggesting here is invalid. Why do you think it's better?